home-server-generator/src/main/resources/mustaches/server/pom.mustache

```diff
@@ -206,6 +202,24 @@
         <version>2.1.1</version>
     </dependency>
 
+    <dependency>
+         <groupId>org.springframework</groupId>
+         <artifactId>spring-beans</artifactId>
+         <version>5.2.8.RELEASE</version>
+    </dependency>
+
+    <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-context</artifactId>
+        <version>2.0.5</version>
```

Comment:
I wondered if `spring-beans` is enough here. But if it isn't the version of `spring-context` should correlate to `spring-beans`

Reply:
Ok, I will try to use it